standup/settings.py

```diff
@@ -235,16 +216,15 @@ def path(*paths):
 CSP_FONT_SRC = (
     "'self'",
 )
-CSP_FORM_ACTION = ("'self'", AUTH0_DOMAIN) if AUTH0_DOMAIN else ("'self'",)
-CSP_CONNECT_SRC = ("'self'", AUTH0_DOMAIN) if AUTH0_DOMAIN else ("'self'",)
+CSP_FORM_ACTION = ("'self'", SITE_URL) if SITE_URL else ("'self'",)
+CSP_CONNECT_SRC = ("'self'", SITE_URL) if SITE_URL else ("'self'",)
```

Comment:
I have no idea if this is right because I'm not sure what these additional bits are for. It's entirely possible this is hold-over from the previous previous auth0 implementation where we were using the JS lock file sources from `AUTH0_DOMAIN`.